ecf/scripts/gdas/atmos/analysis/jgdas_atmos_analysis_calc.ecf

```diff
@@ -6,7 +6,6 @@
 #PBS -l walltime=00:10:00
 #PBS -l select=1:mpiprocs=128:ompthreads=1:ncpus=128
 #PBS -l place=vscatter:exclhost
-#PBS -l hyper=true
```

Comment:
is this because they have enabled hyperthreading by default?

Reply:
yes and based on conversations in the issues meeting it was deemed unnecessary